client/app/resources/i18n/privateDatabase/Messages_fr_FR.xml

```diff
@@ -547,8 +547,12 @@
    <translation id="privateDatabase_tabs_logs_list_no_logs" qtlid="413388">Pas de log pour le moment</translation>
 
 <!-- OOM -->
+   <translation id="privateDatabase_no_oom">Aucun dépassement mémoire à signaler.</translation>
+   <translation id="privateDatabase_warning_oom">{0} dépassements mémoire on eu lieu récemment.</translation>
    <translation id="privateDatabase_modale_oom_title">Dépassements de la mémoire vive</translation>
-   <translation id="privateDatabase_modale_oom_info">Dépassements de la mémoire vive survenus ces {0} derniers jours. Seuls les {1} derniers dépassements sont disponibles.</translation>
+   <translation id="privateDatabase_modale_oom_confirm_button">Commander plus de RAM</translation>
```

Comment:
`mémoire` instead of `RAM` ? Depends on the context

Reply:
This button has the same behaviour as its neighbour : "Changer la quantité de la RAM" so I kept the same wording.

Reply:
I understand but isn't `mémoire` more appropriate than `RAM` ?